src/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/FishboneMoncriefDisk.hpp

```diff
@@ -0,0 +1,310 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <limits>
+#include <pup.h>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/RelativisticEuler/Tags.hpp"
+#include "Evolution/Systems/RelativisticEuler/Valencia/Tags.hpp"  // IWYU pragma: keep
+#include "Options/Options.hpp"
+#include "Utilities/MakeArray.hpp"  // IWYU pragma: keep
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+namespace RelativisticEuler {
+namespace Solutions {
+
+/*!
+ * \brief Fluid disk orbiting a Kerr black hole
```

Comment:
General question about this solution: Is it valid for all time `t` or is it only valid at a specific time? We want to differentiate between analytic *solutions* which are valid at all times, and analytic *data* which is valid only at one instance of time.

Reply:
Just like the Kerr metric is a stationary solution of the Einstein equations, the Fishbone-Moncrief disk is a stationary solution to the relativistic Euler equation, i.e. it's a solution at any time (as it is time independent) Actually, now that you mention this, I forgot to take the `t` argument from the solution. Should I do `/* t */` in the header file, or should I just remove it at all?

Reply:
The analytic data we will use for the accretion disk, however, is not an analytic solution of anything, as it will consist of a Fishbone-Moncrief disk superposed to a magnetic field, plus a perturbation to the pressure given my random noise. That should totally go in an `PointwiseFunctions/AnalyticData` directory (or that's why I'be been thinking.)